storage/client_test.go

```diff
@@ -156,11 +156,15 @@ func (m *multiTestContext) Start(t *testing.T, numStores int) {
 }
 
 func (m *multiTestContext) Stop() {
-	m.clientStopper.Stop()
-	for _, s := range m.stoppers {
+	stoppers := append([]*util.Stopper{m.clientStopper}, m.stoppers...)
+	stoppers = append(stoppers, m.transportStopper)
```

Comment:
It had to be at the end before because the stores couldn't shut down cleanly if the transport was closed out from under them. Now that we have a two-phase quiesce and stop sequence, it may not be necessary (but I'm not sure).